woocommerce/woocommerce-android
# All Product Reviews: Tests

This PR fixes #1392 by adding tests for the `Reviews` component. I was able to add a bunch of unit tests for the ViewModel, and I fixed the `MainNavigatorTest` for the Reviews tab by adding all the Dagger dependencies. I didn't get to add as many tests as I would've liked because there are a lot of broken tests in the instrumentation tests section. I spent a bunch of time trying to iron them all out, but I'll need to wait until full tests are added for Refunds and the new v4 stats code. For now the tests in place are a very good baseline to start with. I dont wnat to hold up getting this feature to production this sprint, so I'm submitting this as-is. 

To Test:

- Run the Unit tests `./gradlew testWasabiDebugUnitTest`
- Run the tests in the `MainNavigatorTests`

All tests should pass
